src/types.ts

```diff
@@ -52,3 +53,45 @@ export enum UserRole {
   REPAIRER = "REPAIRER",
   CLIENT = "CLIENT"
 }
+
+// API Responses on client
+// All dates are returned as ISO strings
+export type EventResponse = {
+  id: string;
+  createdBy: User;
+  /** ISO date string */
+  createdAt: string;
+  /** ISO date string */
+  updatedAt: string;
+  name: string;
+  location: string;
+  description: string;
+  volunteers: User[];
+  eventType: string;
+  disclaimer: string;
+  status: $Enums.EventStatus;
```

Comment:
Nitpick but this could be moved to a separate 'EventStatus' type that extends  `$Enums.EventStatus`